services/zookeeper-operator/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+displayName: ZooKeeper Operator
+description: The ZooKeeper operator deploys Zookeeper 3.6.3 clusters, and uses Zookeeper dynamic reconfiguration to handle node membership.
+category:
+  - Data Services
```

Comment:
@kaiwalyajoshi I guess it's nice if zookeeper is listed alongside Kafka, but i guess zookeeper is really a config management tool?

Reply:
just to close this out, we discussed in standup and decided it should stay in `Data Services`, good call out @cwyl02!